### DISCUSSION TITLE
someone stole my code

### DISCUSSION THREAD

**Discussion by webbrowser11** [community/community#144134] [Code Security]
### Select Topic Area

Question

### Body

a guy named ArtificialXdev stole my code and is denying it but i cant contact github, what do i do?
![Screenshot 2024-11-10 095458](https://github.com/user-attachments/assets/4e9b451c-88b6-44c4-90b4-49000dc329f4)
![Screenshot 2024-11-10 095510](https://github.com/user-attachments/assets/21ce1dd0-972e-467f-9a8a-4f01d8337170)
![Screenshot 2024-11-10 095605](https://github.com/user-attachments/assets/d81ed5fa-cba1-4205-8db2-8ea548a658f9)
![Screenshot 2024-11-10 095632](https://github.com/user-attachments/assets/9e09cd16-7650-4bc9-87d8-47244c8610e9)
![Screenshot 2024-11-10 095652](https://github.com/user-attachments/assets/e4afddb7-9b0b-4679-a515-79fef1133ade)
![Screenshot 2024-11-10 095901](https://github.com/user-attachments/assets/d2e0aa8b-297b-4361-bc2b-4f383a01b12f)
![Screenshot 2024-11-10 101105](https://github.com/user-attachments/assets/1151795a-f728-41a5-bb5d-7520179c493b)
compare to skyOS 2.0 code:
[SkyOS-2.0.zip](https://github.com/user-attachments/files/17692553/SkyOS-2.0.zip)

**Comment by webbrowser11** [comment]
he is also now claiming to be a HACKER and knows what browser and stuff i am on.

**Comment by webbrowser11** [comment]
if someone tracks down his name and last name i am gonna be so thankful

**Comment by ghostinhershell** [comment]
Hi @webbrowser11, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
